### ISSUE TITLE
Won't generate retakes.cfg

### ISSUE THREAD

**Issue by ob12** [splewis/csgo-retakes#69]
I don't know if I'm supposed to make it myself. The problem is I want to change some cvars but I can't find a list of them online, and I've really looked. Please tell me I'm stupid and give me a link to an entry somewhere with the list, I'd be glad. That's all I need, really, the cvar list. The plugin, in every other aspect, works really good. This issue probably has nothing to do with the plugin itself. 

tl;dr I want a list of cvars, a retakes.cfg doesn't generate like it _most likely_ should(?) :-)

**Comment by splewis** [comment]
``cfg/sourcemod/retakes/retakes.cfg`` gets generated by the plugin, but requires:
1. the ``cfg/sourcemod/retakes`` directory already exists (it does if you installed it correctly)
2. the server has write permissions in that directory (if you uploaded the files as root but the servers runs as a lower user this would be an issue)
